rollup.config.js

```diff
@@ -169,23 +174,28 @@ export default [
     ]
   }, {
     ...config,
-    plugins: makePlugins(true),
+    plugins: makePlugins(true, 'ADVANCED_OPTIMIZATIONS'),
```

Comment:
`esm` is the only place where we can do advanced optimizations since `closure` is made for `import/export` rather than `require`.